crates/kitsune_p2p/kitsune_p2p/tests/common/test_legacy_host.rs

```diff
@@ -378,7 +380,10 @@ impl TestLegacyHost {
             .new_seed(tag.into(), None, false)
             .await
             .unwrap();
-        Arc::new(KitsuneAgent(info.ed25519_pub_key.0.to_vec()))
+        let mut pub_key_bytes = info.ed25519_pub_key.0.to_vec();
+        let loc = dht_location(&pub_key_bytes[0..32].try_into().unwrap());
+        pub_key_bytes.extend(&loc);
+        Arc::new(KitsuneAgent::new(pub_key_bytes))
```

Comment:
This was an omission from the test host implementation that agents didn't have a location. Wasn't needed by the existing tests because they all use full arc.